### PR TITLE
Update podman stack

### PR DESCRIPTION
**Pull request by chax**
**Summary**

- Update cni-plugins 1.1.1 -> 1.3.0 
  - [Changelog 1.1.1->1.2.0](https://github.com/containernetworking/plugins/releases/tag/v1.2.0) 
  - [Changelog 1.2.0->1.3.0](https://github.com/containernetworking/plugins/releases/tag/v1.3.0) 
- Update slirp4netns 1.2.0 -> 1.2.2
  - [Changelog 1.2.1](https://github.com/rootless-containers/slirp4netns/releases/tag/v1.2.1)
  - [Changelog 1.2.2](https://github.com/rootless-containers/slirp4netns/releases/tag/v1.2.2)
- Update conmon 2.1.7 -> 2.1.8
  - [Changelog](https://github.com/containers/conmon/releases/tag/v2.1.8)
- Update skopeo 1.12.0 -> 1.13.3
  - [Changelog 1.12.0->1.13.0](https://github.com/containers/skopeo/releases/tag/v1.13.0)
  - Changelog 1.13.0 -> 1.13.3:
    - Build/package improvements
    - Bump some internal dependencies
- Update buildah 1.30.0 -> 1.32.1
  - [Changelog 1.31.0](https://github.com/containers/buildah/releases/tag/v1.31.0)
  - [Changelog 1.32.0](https://github.com/containers/buildah/releases/tag/v1.32.0)
  - [Changelog 1.32.1](https://github.com/containers/buildah/releases/tag/v1.32.1)
- Update podman 4.5.1 -> 4.7.1
  - [Changelog 4.6.0](https://github.com/containers/podman/releases/tag/v4.6.0)
  - [Changelog 4.6.1](https://github.com/containers/podman/releases/tag/v4.6.1)
  - [Changelog 4.6.2](https://github.com/containers/podman/releases/tag/v4.6.1)
  - [Changelog 4.7.0](https://github.com/containers/podman/releases/tag/v4.7.0)
  - [Changelog 4.7.1](https://github.com/containers/podman/releases/tag/v4.7.1)

**Test Plan**

Tried to build new container image with buildah and run it with podman.

**Checklist**

- [x] Packages were built and tested against unstable

### CODE REVIEW

File: packages/b/buildah/package.yml [hunk 1]
```diff
@@ -18,7 +18,6 @@ description: |
     - delete a working container or an image
     - rename a local container
 builddeps  :
-    - pkgconfig(devmapper)
```
Comment (joebonrichie) [inline packages/b/buildah/package.yml:21]: Is this dependency no longer used?

Comment (chax) [reply packages/b/buildah/package.yml:21]: yes, they removed support

Comment (chax) [reply packages/b/buildah/package.yml:21]: https://github.com/containers/buildah/pull/4832

Comment (joebonrichie) [reply packages/b/buildah/package.yml:21]: Nice, thanks

**Review by joebonrichie — APPROVED** [review]

**Pull request merged**